### DISCUSSION TITLE
Docker manifest missing from latest?

### DISCUSSION THREAD

**Discussion by yetdog** [dani-garcia/vaultwarden#3380] [Q&A] [answered]
### Subject of the issue
When trying to pull latest docker image, operation fails

### Steps to reproduce
docker pull vaultwarden/server:latest

### Expected behaviour
The image should pull down

### Actual behaviour
[root@homedocker1 ~]# docker pull vaultwarden/server:latest
Error response from daemon: manifest for vaultwarden/server:latest not found

**Comment by BlackDex** [comment]
Seems to be there for me: https://hub.docker.com/layers/vaultwarden/server/latest/images/sha256-836f746aa2abcb25f36f088d7e728edc8b25fc46ea930051bc9f4639d079db7f?context=explore

**Comment by yetdog** [comment]
Very odd. I pulled down an image from another container just to be sure it wasn't a global issue for me, and it was successful.

[root@homedocker1 ~]# docker pull vaultwarden/server:latest
Error response from daemon: manifest for vaultwarden/server:latest not found
[root@homedocker1 ~]# docker pull vaultwarden/server
Using default tag: latest
Error response from daemon: manifest for vaultwarden/server:latest not found
[root@homedocker1 ~]# docker pull vaultwarden/server:v1.28.0
Error response from daemon: manifest for vaultwarden/server:v1.28.0 not found

**Comment by BlackDex** [comment]
Probably something with your docker setup, or dns resolving maybe?

**Comment by stefan0xC** [comment]
Can you try with `docker pull docker.io/vaultwarden/server:latest`?  Since 1.28.0 you can also try with  `quay.io/vaultwarden/server:latest` and/or `ghcr.io/dani-garcia/vaultwarden:latest`

**Reply by yetdog** [reply]
> Can you try with `docker pull docker.io/vaultwarden/server:latest`? Since 1.28.0 you can also try with `quay.io/vaultwarden/server:latest` and/or `ghcr.io/dani-garcia/vaultwarden:latest`

Thanks for the suggestions. Here's what I see:

[root@homedocker1 ~]# docker pull docker.io/vaultwarden/server:latest
Error response from daemon: manifest for vaultwarden/server:latest not found
[root@homedocker1 ~]# docker pull quay.io/vaultwarden/server:latest
Error response from daemon: missing signature key
[root@homedocker1 ~]# docker pull ghcr.io/dani-garcia/vaultwarden:latest
Pulling repository ghcr.io/dani-garcia/vaultwarden
Error: image dani-garcia/vaultwarden:latest not found

**Reply by BlackDex** [reply]
Strange, i just pulled from all the three repos without issues

**Reply by stefan0xC** [reply]
@yetdog What OS and docker version are you using? Can you also check if your date/time settings are correct?

**Reply by BlackDex** [reply]
And, if you have enough disk space left!

**Reply by yetdog** [reply]
> @yetdog What OS and docker version are you using? Can you also check if your date/time settings are correct?

CentOS Linux release 7.7.1908 (Core)
Docker version 17.05.0-ce, build 89658be

**Reply by BlackDex** [reply]
Hmm, that is a very old system (3 years old, and not supported anymore i think) and old docker version.

**Reply by stefan0xC** [reply]
I would also recommend to at least update to a newer docker version. See https://docs.docker.com/engine/install/centos/ for more information.

**Comment by yetdog** [comment]
So here's what my messages log says. Sorry I didn't think to look/include this in the first place"

Mar 27 09:05:30 homedocker1 dockerd: time="2023-03-27T09:05:30.953227158-07:00" level=info msg="Attempting next endpoint for pull after error: manifest unknown: OCI index found, but Accept header does not support OCI indexes"
Mar 27 09:05:31 homedocker1 dockerd: time="2023-03-27T09:05:31.297682579-07:00" level=error msg="Not continuing with pull after error: Error: image dani-garcia/vaultwarden:latest not found"

**Comment by yetdog** [comment, marked as answer]
Thanks all - i updated Docker and all is well.
